_episodes_rmd/03-seeking-help.Rmd

```diff
@@ -62,9 +62,9 @@ To seek help on special operators, use quotes:
 ## Getting help on packages
 
 Many packages come with "vignettes": tutorials and extended example documentation.
-Without any arguments, `vignette()` will list all vignettes for all installed packages;
-`vignette(package="package-name")` will list all available vignettes for
-`package-name`, and `vignette("vignette-name")` will open the specified vignette.
+Without any arguments, `vignette()` and `vignette("vignette-name")` will open the 
```

Comment:
I think this is a typo? On its own, `vignette()` doesn't open any vignettes, it lists all available for installed packages. In the proposed change, this is not described.